.github/workflows/wheels.yml

```
@@ -199,3 +199,50 @@ jobs:
           source ci/upload_wheels.sh
           set_upload_vars
           upload_wheels
+
+  # Build Pyodide wheels and upload them to Anaconda.org
+  # NOTE: this job is similar to the one in unit-tests.yml except for the fact
+  # that it uses cibuildwheel instead of a standard Pyodide xbuildenv setup.
+  build_pyodide_wheels:
+    name: Build wheel for cp312-pyodide_wasm32

```

```suggestion
    name: Build wheel for pyodide_wasm32

```

Because cibuildwheel will take care of future Python versions?

Also, so far this wheel will only be available for Python 3.12?

---

Your suggestion makes sense, thanks – addressed in 3fb08328d4bac43f8b7f78e65ab35fa0b7772163. Yes, Pyodide provides just one Python version under its belt with a major release, and therefore so does `cibuildwheel`.